src/api/rest/server/resource/k8scluster.go

```diff
@@ -700,3 +700,38 @@ func RestRecommendK8sNode(c echo.Context) error {
 	content, err := infra.RecommendK8sNode(nsId, *u)
 	return common.EndRequestWithLog(c, err, content)
 }
+
+// RestPostCmdK8sCluster godoc
+// @ID PostCmdK8sCluster
+// @Summary Send a command to specified Container in K8sCluster
+// @Description Send a command to specified Container in K8sCluster
```

Comment:
현재 한정적인 기능으로 API를 제공하는 상황이므로, 아래 정도의 API 코멘트를 제공하면 좋을 것 같습니다. :)

```suggestion
// @Description Send a command to specified Container in K8sCluster
// @Description [note] This feature is not intended for general use 
// @Description This API is provided as an exceptional and limited function for specific purposes such as migration. 
// @Description Kubernetes resource information required as input for this API is not currently provided, and its availability in the future is uncertain.
```

Reply:
코멘트 고맙습니다.
반영하였습니다.